### PR TITLE
[stable/prometheus-mysql-exporter] to enable/disable collector flags …

### PR DESCRIPTION
**Pull request by kilhyunjun**
What this PR does / why we need it:

to enable/disable mysql-exporter collector flags from the value file.

Signed-off-by: Kilhyun Jun <Kilhyun@gmail.com>

@Juanchimienti 

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prometheus-mysql-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -32,6 +32,11 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+{{- with .Values.collectors}}
+          command: ["/bin/mysqld_exporter",
```
Comment (paulczar) [inline stable/prometheus-mysql-exporter/templates/deployment.yaml:36]: should we pass these in as `args` vs in the command itself ?

File: stable/prometheus-mysql-exporter/values.yaml [hunk 2]
```diff
@@ -70,3 +70,10 @@ cloudsqlproxy:
       "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
       "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/user%40project.iam.gserviceaccount.com"
     }'
+
+# mysql exporter collectors https://github.com/prometheus/mysqld_exporter#collector-flags
+collectors:
```
Comment (paulczar) [inline stable/prometheus-mysql-exporter/values.yaml:75]: this should be an array to pass to args right ?  then in the deployment could use  `args: {{ args | toyaml }}`

```
args:
  - "--collect.global_status"
  - "--collect.global_variables"
```
or could even do

```
collectors:
  - global_status
  - global_variables
```

and use a helper to add the `--collect.` to the start of each one ?

**Review by paulczar — CHANGES_REQUESTED** [review]

**Comment by estahn** [comment]
@paulczar What's the status with this? Would you accept a new PR with the changes you request? I want to get the process list so I can alert on long running queries.

**Pull request closed without merging**